pom.xml

```diff
@@ -0,0 +1,213 @@
+    <properties>
+        <siddhi.version>5.1.5</siddhi.version>
+        <siddhi.version.range>[5.0.0,6.0.0)</siddhi.version.range>
+        <commons.logging.version>1.1.1</commons.logging.version>
+        <commons.logging.version.range>[1.1.1,2.0)</commons.logging.version.range>
+        <testng.version>6.11</testng.version>
+        <jacoco.plugin.version>0.7.9</jacoco.plugin.version>
+        <jacoco.ant.version>0.7.9</jacoco.ant.version>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <siddhi.version>5.1.5</siddhi.version>
+        <siddhi.bundle.version>5.1.5</siddhi.bundle.version>
+        <siddhi.version.range>[5.0.0, 6.0.0)</siddhi.version.range>
+        <carbon.analytics.common.version>6.0.73</carbon.analytics.common.version>
+        <siddhi.execution.string.version>5.0.5</siddhi.execution.string.version>
+        <slf4j.version>1.7.12</slf4j.version>
+        <slf4j.logging.package.import.version.range>[1.7.1, 2.0.0)
+        </slf4j.logging.package.import.version.range>
+        <log4j.version>1.2.17.wso2v1</log4j.version>
+        <jaxb.version>2.2.11</jaxb.version>
+        <javax.annotation.version>1.1.1</javax.annotation.version>
+    </properties>
+
+    <scm>
+        <connection>scm:git:https://github.com/wso2-extensions/siddhi-store-cosmosdb.git</connection>
+        <url>https://github.com/wso2-extensions/siddhi-store-cosmosdb.git</url>
+        <developerConnection>scm:git:https://github.com/wso2-extensions/siddhi-store-cosmosdb.git</developerConnection>
+        <tag>HEAD</tag>
+    </scm>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.siddhi</groupId>
+                <artifactId>siddhi-core</artifactId>
+                <version>${siddhi.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.siddhi</groupId>
+                <artifactId>siddhi-query-api</artifactId>
+                <version>${siddhi.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>commons-logging</groupId>
+                <artifactId>commons-logging</artifactId>
+                <version>${commons.logging.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.testng</groupId>
+                <artifactId>testng</artifactId>
+                <version>${testng.version}</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.jacoco</groupId>
+                <artifactId>org.jacoco.ant</artifactId>
+                <version>${jacoco.ant.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-lang3</artifactId>
+                <version>3.3.2</version>
```

Comment:
@Tikiri16 shall we move these to properties section?

Reply:
Moved.